doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,134 @@
+# Support LibSVM Format Column in SQLFlow
+
+## Definition
+
+### LibSVM Format File
```

Comment:
Can put this to the last section "Appendix".

Reply:
I prefer to put this into the first section to make readers more clear about the important concepts in the design doc.

Reply:
It doesn't help readers to understand the document. It is confusing. SQLFlow doesn't read files in the LibSVM format at all.